apis/Google.Bigquery.V2/Google.Bigquery.V2.Snippets/BigqueryClientSnippets.cs

```diff
@@ -637,28 +638,622 @@ public void ParameterizedQuery_PositionalParameters()
+            BigqueryClient.Create(projectId).CreateDataset(datasetId);
+            // Snippet: DeleteDatasetAsync(string,DeleteDatasetOptions,*)
+            BigqueryClient client = await BigqueryClient.CreateAsync(projectId);
+            await client.DeleteDatasetAsync(datasetId);
+            // End snippet
+
+            IPagedEnumerable<DatasetList, BigqueryDataset> datasets = client.ListDatasets();
+            List<string> ids = datasets.Select(ds => ds.Reference.DatasetId).ToList();
+            Assert.DoesNotContain(datasetId, ids);
+        }
+
+        [Fact]
+        public async Task ListProjectsAsync()
+        {
+            // Snippet: ListProjectsAsync(*,*)
+            BigqueryClient client = BigqueryClient.Create("irrelevant");
+            IPagedAsyncEnumerable<ProjectList, CloudProject> projects = client.ListProjectsAsync();
+            await projects.ForEachAsync(project =>
+            {
+                Console.WriteLine($"{project.ProjectId}: {project.FriendlyName}");
+            });
+            // End snippet
+            Assert.Contains(_fixture.ProjectId, await projects.Select(p => p.ProjectId).ToList());
+        }
+
+        [Fact]
+        public async Task ParameterizedQueryAsync_NamedParameters()
+        {
+            // Sample: ParameterizedQueryNamedParametersAsync
+            // Additional: ExecuteQueryAsync(BigqueryCommand,*,*)
+            BigqueryClient client = await BigqueryClient.CreateAsync(_fixture.ProjectId);
+            BigqueryTable table = await client.GetTableAsync(_fixture.GameDatasetId, _fixture.HistoryTableId);
+            BigqueryCommand command = new BigqueryCommand($"SELECT player, score, level FROM {table} WHERE score >= @score AND level >= @level");
+            // Note: could also use a collection initializer to populate the parameters.
+            command.Parameters.Add("level", BigqueryParameterType.Int64).Value = 2;
+            command.Parameters.Add("score", BigqueryParameterType.Int64).Value = 1500;
+            BigqueryQueryJob queryJob = await client.ExecuteQueryAsync(command);
+            queryJob = await queryJob.PollUntilCompletedAsync();
+            IAsyncEnumerable<BigqueryRow> queryResults = queryJob.GetRowsAsync();
+            await queryResults.ForEachAsync(row =>
+            {
+                Console.WriteLine($"Name: {row["player"]}; Score: {row["score"]}; Level: {row["level"]}");
+            });
+            // End sample
+        }
+
+        [Fact]
+        public async Task ParameterizedQueryAsync_PositionalParameters()
+        {
+            /// Sample: ParameterizedQueryPositionalParameters
+            BigqueryClient client = BigqueryClient.Create(_fixture.ProjectId);
+            BigqueryTable table = client.GetTable(_fixture.GameDatasetId, _fixture.HistoryTableId);
+            BigqueryCommand command = new BigqueryCommand($"SELECT player, score, level FROM {table} WHERE score >= ? AND level >= ?");
```

Comment:
[Not for this PR]
I still think it'd be nice to be able to write something like:
new BigqueryCommand($"SELECT player, score, level FROM {table} WHERE score >= ? AND level >= ?", 1500, 2);